megamek/src/megamek/client/bot/princess/FireControl.java

```diff
@@ -1846,13 +1846,25 @@ FiringPlan getFullFiringPlan(final Entity shooter,
         		continue;
         	}
         	
-        	
-            final double toHitThreshold = ammoConservation.get(weapon);
-            final WeaponFireInfo shoot = buildWeaponFireInfo(shooter, target, weapon, game, false);
+        	final double toHitThreshold = ammoConservation.get(weapon);
```

Comment:
This line is tab spaced, and it looks like some other parts of this file are too.

Reply:
Don't have access to my dev environment right now, so we'll get this one next time.